resources/util.go

```diff
@@ -133,3 +135,35 @@ func LowerStructFieldNameToFieldName(res Res) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+// GetUID returns the UID of an user
```

Comment:
Please make the doc comments into proper sentences ending with a period for example and formatted to 80 chars please.